src/main/java/seedu/recipe/logic/commands/FindCommand.java

```diff
@@ -14,15 +16,17 @@ public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all recipes whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a "
-            + "list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all recipes whose property contain any of "
```

Comment:
Might be more readable if we move everything after `COMMAND_WORD` to the next line